appendices/versions.txt

```diff
@@ -188,37 +188,70 @@ number of key extensions into the core API:
+the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
+is optional.
+Support for this feature is defined by
+slink:VkPhysicalDevice16BitStorageFeatures::pname:storageBuffer16BitAccess
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_KHR_sampler_ycbcr_conversion`
 
 If the `<<VK_KHR_sampler_ycbcr_conversion>>` extension is not supported,
 support for the <<features-samplerYcbcrConversion,samplerYcbcrConversion>>
 feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceSamplerYcbcrConversionFeatures::pname:samplerYcbcrConversion
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
-=== Differences relative to `VK_KHR_16bit_storage`
+=== Differences relative to `VK_KHR_shader_draw_parameters`
 
-If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
-the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
-is optional.
+If the `<VK_KHR_shader_draw_parameters>>` extension is not supported,
+support for the <<features-shaderDrawParameters,shaderDrawParameters>>
+feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceShaderDrawParametersFeatures::pname:shaderDrawParameters
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:shaderDrawParameters
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
+
+=== Differences relative to `VK_KHR_variable_pointers`
+
+If the `<<VK_KHR_variable_pointers>>` extension is not supported,
+support for the <<features-variablePointersStorageBuffer,
+pname:variablePointersStorageBuffer>> feature is optional.
+Support for this feature is defined by
+slink:VkPhysicalDeviceVariablePointersFeatures::pname:variablePointersStorageBuffer
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:variablePointersStorageBuffer
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
+
+=== Additional Vulkan 1.1 Feature Support
+
+[[versions-1.1-new-features]]
+In addition to the promoted extensions described above, Vulkan 1.1 added
+support for:
+
+  * The <<shaders-group-operations, group operations>> and
+    <<shaders-scope-subgroup, subgroup scope>>.
+  * The <<memory-protected-memory, protected memory>> feature.
+  * A new command to enumerate the instance version:
+    flink:vkEnumerateInstanceVersion.
+  * The <<features-shaderDrawParameters,shaderDrawParameters>> feature query
```

Comment:
either we should remove this line or add all the new added query features... I prefer the first as this seems like duplicate info

Reply:
There are no other added feature bools. All the others are promoted, and protected memory is mentioned above.